src/connector01917/DBConnector.java

```diff
@@ -0,0 +1,96 @@
+package connector01917;
+
+import daointerfaces01917.DALException;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.DriverManager;
+import java.sql.Statement;
+
+/**
+ * Created by AndersWOlsen on 29-05-2017.
+ */
+public class DBConnector implements IConnector {
+    private Connection connection;
+    private Statement statement; // TODO: Make this a prepared statement, in a nice way
+    private boolean connectionIsOpen;
+
+    // private String host, database, username, password; // TODO: Used in CDIO1, I don't believe we need it?
```

Comment:
We definitely need this while testing, unless you want the info passed by the constructor